blazor/dropdown-list/data-binding.md

```diff
@@ -9,66 +9,890 @@ documentation: ug
+
+{% tabs %}
+{% highlight razor %}
+@using Syncfusion.Blazor.Data
+@using Syncfusion.Blazor.DropDowns
+
+<SfDropDownList TValue="string" TItem="Order" Placeholder="Select a Country">
+    <SfDataManager Url="api/Default" Adaptor="Adaptors.WebApiAdaptor" CrossDomain="true"></SfDataManager>
+    <DropDownListFieldSettings Text="ShipCountry" Value="OrderID"></DropDownListFieldSettings>
+</SfDropDownList>
+
+@code{
+    public class Order
+    {
+        public int? OrderID { get; set; }
+        public string ShipCountry { get; set; }
+    }
+}
+{% endtabs %}
+{% endhighlight razor %}
+
+## Adding New Items
+
+You can add the new item in the popup with help of using [AddItemsAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_AddItemsAsync_System_Collections_Generic_IEnumerable__0__System_Nullable_System_Int32__) public method. This method will add a mentioned item in the DropDownList popup without affecting the data source items.
+
+{% tabs %}
+{% highlight razor %}
+@using Syncfusion.Blazor.DropDowns
+@using Syncfusion.Blazor.Buttons
+
+<div>
+<SfDropDownList @ref="ddlObj" TValue="string" TItem="Games" Width="300px" Placeholder="Select a game" DataSource="@LocalData">
+  <DropDownListFieldSettings Value="ID" Text="Game"></DropDownListFieldSettings>
+</SfDropDownList>
+</div>
+<div>
+<SfButton Content="Click to add a new item" OnClick="OnBtnClick"></SfButton>
+</div>
+
+@code {
+    SfDropDownList<string, Games> ddlObj;
+    public class Games
+    {  
+        public string ID { get; set; }
+        public string Game { get; set; }
+    }
+    List<Games> LocalData = new List<Games> {
+    new Games() { ID= "Game1", Game= "American Football" },
+    new Games() { ID= "Game2", Game= "Badminton" },
+    new Games() { ID= "Game3", Game= "Basketball" },
+    new Games() { ID= "Game4", Game= "Cricket" },
+    new Games() { ID= "Game5", Game= "Football" },
+    new Games() { ID= "Game6", Game= "Golf" },
+    new Games() { ID= "Game7", Game= "Hockey" },
+    new Games() { ID= "Game8", Game= "Rugby"},
+    new Games() { ID= "Game9", Game= "Snooker" },
+  };
+    public async Task OnBtnClick()
+    {
+       await this.ddlObj.AddItemsAsync(new List<Games> { new Games() { ID = "Game11", Game = "Tennis" } });
+    }
+}
+{% endtabs %}
+{% endhighlight razor %}
+
+The output will be as follows.
+
+![Blazor DropDownList with Add Item](./images/blazor_dropdown_bind_addItem.png)
+
+## OnActionBegin Event
+
+The [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.DropDownListEvents-2.html#Syncfusion_Blazor_DropDowns_DropDownListEvents_2_OnActionBegin) event will trigger before fetching data from the remote server. By using the respective event we can perform required things when the component before fetching data in it. The below code example demosntrates the usage of OnActionBegin event.
+
```

Comment:
@BerlyChristopher94 : Don't use we in the document

![image](https://user-images.githubusercontent.com/29191202/157014001-26fd6473-bf4a-408e-876e-ad3865a6e43a.png)